api/v1/db_types.go

```diff
@@ -12,8 +12,20 @@ import (
 	"gorm.io/gorm/schema"
 )
 
+const (
+	SQLServerType = "sqlserver"
+	PostgresType  = "postgres"
+	SqliteType    = "sqlite"
```

Comment:
These are capital because we are exporting them too ?

Reply:
There are other places where they might be used... for example the pkg.Properties is in a different package but we need those values there